plonky2/src/gates/coset_interpolation.rs

```diff
@@ -713,63 +713,134 @@ mod tests {
     use p3_field::AbstractField;
     use p3_goldilocks::Goldilocks;
     use plonky2_field::polynomial::PolynomialValues;
+    use plonky2_field::{GOLDILOCKS_EXTENSION_FIELD_DEGREE, GOLDILOCKS_NUM_HASH_OUT_ELTS};
     use plonky2_util::log2_strict;
 
     use super::*;
     use crate::field::types::Sample;
     use crate::gates::gate_testing::{test_eval_fns, test_low_degree};
-    use crate::hash::hash_types::{HashOut, GOLDILOCKS_NUM_HASH_OUT_ELTS};
+    use crate::hash::hash_types::HashOut;
     use crate::plonk::config::{GenericConfig, PoseidonGoldilocksConfig};
 
     #[test]
     fn test_degree_and_wires_minimized() {
         const NUM_HASH_OUT_ELTS: usize = GOLDILOCKS_NUM_HASH_OUT_ELTS;
-        let gate = <CosetInterpolationGate<Goldilocks, 2>>::with_max_degree(3, 2);
+        let gate = <CosetInterpolationGate<Goldilocks, GOLDILOCKS_EXTENSION_FIELD_DEGREE>>::with_max_degree(3, 2);
```

Comment:
For consistency, we should create a `const EXTENSION_FIELD_DEGREE` (or `D` for brevity) and reuse it in the test, in the same way we do for `NUM_HASH_OUT_ELTS` in the line above.

Reply:
Done